cypress/support/helpers/checkA11y.js

```diff
@@ -18,7 +18,7 @@ const logA11yViolations = violations => {
       impact,
       description,
       occurrences: nodes.length,
-      paths: nodes.map(({ target }) => target).join('\n'),
+      paths: nodes.map(({ target }) => target).join(', '),
```

Comment:
Just to check the paths value is meant to be in this format `paths:  "foo,  bar"`?

Reply:
Yep!